mvpa2/tests/test_transerror.py

```diff
@@ -673,6 +673,26 @@ def test_confusionmatrix_nulldist(self):
                                       cvnp[(np.array([0,1]), np.array([1,0]))])
 
 
+def test_plotting_subset():
```

Comment:
so if we do allow (optionally) the subset to be visualized -- then yeah, worth having this test

our use case though is different -- it is when we provide labels (to order) which has more labels than it is known to the ca.stats.labels (like we include "brain" in providing the order, but then we had no "brain" label within our dataset we worked on)